src/components/Calendar/RangePicker.story.js

```diff
@@ -0,0 +1,32 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info';
+
+import withTests from '../../util/withTests';
+import { RangePicker } from '.';
+
+class CalendarStoryPicker extends Component {
```

Comment:
I think you could also use our `State` component here, for consistency.

Reply:
You know, writing them this way might actually fix the problem with the automatically generated docs? Perhaps this is better than the State component or withState.

Reply:
Switched to using `recompose` as discussed.